index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">
+          <img src="images/Union.png" alt="menubar" />
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section class="Port">
+        <div hreh="#" class="Menu"></div>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), kindly use Flexbox in this section in order to lay out the elements on the page 🛠👍